server/src/main/java/org/apache/iotdb/db/engine/storagegroup/VirtualStorageGroupProcessor.java

```diff
@@ -263,7 +263,7 @@
   // DEFAULT_POOL_TRIM_INTERVAL_MILLIS
   private long timeWhenPoolNotEmpty = Long.MAX_VALUE;
 
-  private LastFlushTimeManager lastFlushTimeManager = new LastFlushTimeManager();
+  private ILastFlushTimeManager lastFlushTimeManager = new LastFlushTimeManager();
```

Comment:
Line 414 created a new LastFlushTimeManager, so we don't need to create new here?

Reply:
fixed~